src/test/kotlin/org/rustSlowTests/cargo/runconfig/buildtool/CargoBuildTest.kt

```diff
@@ -23,8 +23,6 @@ import org.rustSlowTests.cargo.runconfig.RunConfigurationTestBase
 
 abstract class CargoBuildTest : RunConfigurationTestBase() {
 
-    override fun shouldRunTest(): Boolean = System.getenv("CI") == null
```

Comment:
These tests were disabled because they are flaky. I think this problem still exists.